src/Sarif/Visitors/InsertOptionalDataVisitor.cs

```diff
@@ -39,24 +50,9 @@ public override Run VisitRun(Run node)
                 visitor.VisitRun(node);
             }
 
-            if (node.Files != null)
```

Comment:
>null [](start = 30, length = 4)

Why do we no longer visit the `run.files` dictionary?

Reply:
Oh! because `base.VisitRun` does it. You're not doing anything derived-class-specific here; you're just doing the deep dive that the base class already does. Good catch!

---
In reply to: [214519859](https://github.com/Microsoft/sarif-sdk/pull/1062#discussion_r214519859) [](ancestors = 214519859)